src/backend/mod.rs

```diff
@@ -60,6 +60,11 @@ pub enum ClearType {
     UntilNewLine,
 }
 
+pub struct WindowSize {
+    pub columns_rows: Size,
+    pub window_pixels: Size,
```

Comment:
nit: what do you think about naming this just `pixels` given that the struct is named **`Window`**`Size`?

Reply:
I think there should be a doc string here too, right? Maybe describing when someone might want to use this.